### PR TITLE
Yzhbankov/session scan

### PR DESCRIPTION
**Pull request by yzhbankov**

### CODE REVIEW

File: apps/data-scanner/lib/models/session/Client.mjs [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        await this.#api.connect();
+    }
+
+    /**
+     * @method
+     * @param {Object} command - Telegram command to be invoked
+     * @returns {Promise<Object>}
+     */
+    async invoke(command) {
+        return this.#api.session.invoke(command);
```
Comment (whiteadam2) [inline apps/data-scanner/lib/models/session/Client.mjs:50]: It seems here must be
this.#api.invoke(command);

File: apps/data-scanner/lib/models/session/Session.mjs [hunk 2]
```diff
@@ -0,0 +1,56 @@
+import { Client } from './Client.mjs';
+import { writeToFile, readDir, readSqlite } from '../../utils/index.mjs';
+
+export class Session {
```
Comment (whiteadam2) [inline apps/data-scanner/lib/models/session/Session.mjs:4]: I suggest rename Session to Pool or SessionPool or at least Sessions

Comment (yzhbankov) [reply apps/data-scanner/lib/models/session/Session.mjs]: done

File: apps/data-scanner/lib/models/session/Client.mjs (hunk 1 above)
Comment (yzhbankov) [reply apps/data-scanner/lib/models/session/Client.mjs]: done

**Review by whiteadam2 — APPROVED** [review]

**Pull request merged**